src/components/pages/rules/rulesGrid/rulesGrid.scss

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+
+.btn.rule-status-btn {
+  .btn-icon svg {
+    fill: transparent;
+    @include square-px-rem(24px);
+    @include rem-fallback(padding-right, 10px);
+  }
+
+  @include themify($themes) {
+    .cls-1 { stroke: themed('colorBtnSvgFillActive'); }
+
+    .cls-2 {
+      fill: themed('colorContextMenuBtnSvgFill');
+      stroke: themed('colorContextMenuBtnSvgFill');
+    }
+
+    &:hover {
```

Comment:
Two things:
1) If you don't need `.btn-icon` here you probably don't need it above. Please double check.
2) No need to nest in this case since you only have one property `&:hover svg { ... }`